assets/js/trivyScans.js

```diff
@@ -6,61 +6,65 @@ jQuery(function () {
   function transformData(data) {
     const transformed = []
     data.forEach(item => {
-      const summary = item.scan_summary?.summary || {}
-      const scanResult = item.scan_summary?.scan_result || {}
-      const cveIDs = []
-
-      // Extract CVE VulnerabilityID from os-pkgs and lang-pkgs
-      scanResult['os-pkgs']?.forEach(pkg => {
-        if (pkg.VulnerabilityID && pkg.PrimaryURL) {
-          cveIDs.push(`<a href="${pkg.PrimaryURL}" target="_blank">${pkg.VulnerabilityID}</a>`)
-        } else if (pkg.VulnerabilityID) {
-          cveIDs.push(pkg.VulnerabilityID)
-        }
-      })
+      if (item.scan_status === 'Succeeded') {
```

Comment:
So just a little tidy to remove the `if` as it's all dependent on that check:

```
data
  .filter(item => item.scan_status === 'Succeeded')
  .forEach(item => {
```
Then you can remove the `if` 👍